components/RoomInfo.vue

```diff
@@ -18,14 +18,13 @@ let toggleStream: ToggleStreamFunction | undefined
 let leaveRoom: LeaveRoomFunction | undefined
 let toggleChat: ToggleChatFunction | undefined
 
+leaveRoom = inject<LeaveRoomFunction>("leaveRoom")
+toggleChat = inject<ToggleChatFunction>("ToggleChat")
+
 if (props.isSfu) {
 	toggleStream = inject<ToggleStreamFunction>("handleToggleStreamSfu")
-	leaveRoom = inject<LeaveRoomFunction>("leaveRoomSfu")
```

Comment:
not needed since both pages use the same leaveRoom and ToggleChat function but different streaming function